charts/incubator/linkwarden/values.yaml

```diff
@@ -0,0 +1,70 @@
+image:
+  repository: ghcr.io/linkwarden/linkwarden
+  pullPolicy: IfNotPresent
+  tag: v2.2.1@sha256:c948cdcf683ec7b6c634eb2a0e184e270e167401d41ef8e82f7ff64c61f39f9c
+
+config:
+  pagination_take_count: 20
+  autoscroll_timeout: 30
+  disable_registration: false
+  archive_limit: 5
+  smtp:
+    enabled: false
+    from: ""
+    server: ""
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 2985
+        targetPort: 3000
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            DATABASE_URL:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: std
+            POSTGRES_PASSWORD:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            NEXTAUTH_SECRET:
+              secretKeyRef:
+                name: linkwarden-secrets
+                key: NEXTAUTH_SECRET
+            NEXTAUTH_URL: "http://localhost:3000/api/v1/auth"
```

Comment:
Should this always point to localhost?

Reply:
It could probably be changed to the host:port or ingress, I'm sure sure what value would be used to determine the host or if it's ingress

Reply:
https://github.com/linkwarden/linkwarden/issues/269#issuecomment-1789985391

Reply:
https://github.com/truecharts/charts/blob/2dba0faff0449eac2c45976105c6084829c3dbc7/charts/stable/nextcloud/templates/_configmap.tpl#L4C1-L4C1

This should contain the ingress host

Reply:
For whatever reason that NEXTAUTH_URL_INTERNAL isn't taking effect.... or else that would have removed the need for targetPort, and currently ingress doesn't work

Reply:
I've at least added a question for NEXTAUTH_URL so the user can put their public URL to fix the logout issue. Tried with and without ingress but NEXTAUTH_URL_INTERNAL doesn't change, but I think there might be a weird problem with my test system and ingress so if you are able to test that out with ingress let me know

Reply:
Would it be fine to just remove NEXTAUTH_URL_INTERNAL and keep targetPort for now?